pkgs/development/interpreters/python/cpython/3.7/darwin-libutil.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/Modules/posixmodule.c b/Modules/posixmodule.c
```

Comment:
How is this related to the rest of the changes?

Reply:
python3 is now required for building glibc. When python is built in bootstrap phase fetchpatch causes infinite recursion. I've solved this by including patch in nixpkgs source. Other ideas are welcomed.

Reply:
in this case https://github.com/NixOS/nixpkgs/pull/66762 may come in handy

Reply:
Well, run-time closure reduction isn't important there, but reducing the build-time closure could speed up bootstrapping (though that by itself doesn't sound a big deal to me).